### ISSUE TITLE
Determine if Pickling is a good idea for persistant classifier storage or not

### ISSUE THREAD

**Issue by bewakes** [eoglethorpe/DEEPL#1]
One problem arose: Classifier object has an attribute feature_function which is a function and can't be pickled unless it is defined at the top level of module.

**Comment by eoglethorpe** [comment]
Yeah, don’t use this file, it’s old. Use export.py which exports to csv and
is more practical for now
